ModelBundle/DataFixtures/MongoDB/DemoContent/HomeDataGenerator.php

```diff
@@ -2,141 +2,58 @@
      */
     protected function generateNodeDe()
     {
-        $htmlContent = <<<EOF
-<div class='content2'>
-    <h1>Open-Orchestra</h1>
-    <p>Offene Orchestra ist ein leistungsfähiges Web-Integrationsplattform zur Beschleunigung der Bau von mehrjährige digitale Ökosysteme.
-     Diese Lösung von Interakting Erfahrung in der Entwicklung der internationalen Plattformen ist verfügbar unter der Open Source Lizenz.</p>
-    <p>In strikter Übereinstimmung mit den Standards und Best Practices Framework entwickelt auf Symfony2 und MongoDB.
-    Offene Orchestra ist schnell, sehr anpassungsfähig und erweiterbar, Multi-Site und Multi-Geräte.</p>
-    <p>Offene Orchestra CMS bietet erweiterte Funktionen und SI internen und externen Integrationskomponenten
-    in hohem Maße konfigurierbar, modular, hohen Belastungen und Sicherheit zugeschnitten.</p>
-    <p>Eine gezielte Lösung :
-    <ul>
-        <li>Projekt, wo «Erfahrung», ob Kunde, Mitarbeiter, Partner oder Händler ist der Kern des Problems.</li>
-        <li>Internationale Projekt Skaleneffekte erfordern.</li>
-        <li>Komplexe Projekte, bei denen interne Informationssysteme und Partner sind hoch beanspruchte.</li>
-        <li>Projekt, dessen Ziel ist es, digitale Ökosysteme (E-Commerce, Kommunikation, Referenz, Selbst-Pflege, Mobilität, Vertrieb, ...),
-        die mit den funktionalen und technologischen Synergien aufzubauen.</li>
-    </ul></p>
-    <p>Unser versprechen : « Skaleneffekte und die Bündelung von Investitionen für eine konsistente Web-Erfahrung auf allen Kanälen Fixed Mobile,
-    Tablet, TV, Poller ... »</p>
-</div>
-EOF;
         $routePattern = "de";
         $language = "de";
 
-        return $this->generateNodeGlobal($htmlContent, $language, $routePattern);
+        return $this->generateNodeGlobal($language, $routePattern);
     }
 
     /**
-     * @param string $htmlContent
      * @param string $language
      * @param string $routePattern
      *
      * @return Node
      */
-    protected function generateNodeGlobal($htmlContent, $language, $routePattern)
+    protected function generateNodeGlobal($language, $routePattern)
     {
-        $siteHomeBlock0 = new Block();
-        $siteHomeBlock0->setLabel('Wysiwyg');
-        $siteHomeBlock0->setComponent(TinyMCEWysiwygStrategy::NAME);
-        $siteHomeBlock0->setAttributes(array(
-            "htmlContent" => $htmlContent
-        ));
-        $siteHomeBlock0->addArea(array('nodeId' => 0, 'areaId' => 'mainContentArea1'));
-
-        $siteHomeArea0 = $this->createHeader();
-        $siteHomeArea4 = $this->createArea('Main content area 1', 'mainContentArea1', 'main-content-area1');
-        $siteHomeArea4->addBlock(array('nodeId' => 0, 'blockId' => 1));
-        $siteHomeArea5 = $this->createModuleArea();
-        $siteHomeArea3 = $this->createMain(array($siteHomeArea4, $siteHomeArea5));
-        $siteHomeArea6 = $this->createFooter();
-
+        $siteHomeArea0 = $this->createArea('Header','header','header','h');
```

Comment:
I'm agree with you but i did that for two reason 
-first because i need to reuse this in functional fixtures to add the header's blocks.
-two because for now user can't reorder area so if user decide to use this page he can just add the main and footer  page manually